workers/check/dataset.py

```diff
@@ -113,7 +119,7 @@ def callback(client_state, channel, method, properties, input_message):
         )
         nack(client_state, channel, delivery_tag, requeue=False)
 
-    ack(client_state, channel, delivery_tag)
+        ack(client_state, channel, delivery_tag)
```

Comment:
We can only ack/nack a message once. This is a bug I introduced in https://github.com/open-contracting/pelican-backend/commit/768ab97ac8be8ccf3bbbb6a899ab821d07c90cfc?w=1#diff-b1dbd0590713e8c049046818a70aa5b98ee878484f20fecab6fb7259fe0efa84. I'll fix.